apps/core/src/PostHogAnalyticsProvider.tsx

```diff
@@ -5,20 +5,20 @@ import { useFeature } from '@growthbook/growthbook-react';
 import { PostHogProvider } from 'posthog-js/react';
 import { type ReactNode } from 'react';
 
-import { GROWTHBOOK_FEATURES } from '~/utils/growthbook';
-
 type PostHogProviderProps = {
+    projectApiKey: string;
     children: ReactNode;
 };
 
-export function PostHogAnalyticsProvider({ children }: PostHogProviderProps) {
-    const { on: isEnabled } = useFeature(
-        GROWTHBOOK_FEATURES.EXPLORER_POSTHOG_ANALYTICS
-    );
+export function PostHogAnalyticsProvider({
```

Comment:
throwing this in `@mysten/core` for now, if we go with PostHog we'll clean up the Growthbook feature and probably make `isEnabled` a prop for turning PostHog off during development. I added the `isEnabled` check inside the component here just for the sake of simplicity